core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/DataInstructionRepository.java

```diff
@@ -71,20 +72,11 @@
         RandomUUID uuid = new RandomUUID();
         dataInstructions.put(uuid.getKeyword(), uuid);
 
-        Addition addition = new Addition();
-        dataInstructions.put(addition.getKeyword(), addition);
+        JsonPathTraversal jsonPathTraversal = new JsonPathTraversal();
+        dataInstructions.put(jsonPathTraversal.getKeyword(), jsonPathTraversal);
 
-        Substraction substraction = new Substraction();
-        dataInstructions.put(substraction.getKeyword(), substraction);
-
-        Division division = new Division();
-        dataInstructions.put(division.getKeyword(), division);
-
-        Exponentiation exponentiation = new Exponentiation();
-        dataInstructions.put(exponentiation.getKeyword(), exponentiation);
-
-        Multiplication multiplication = new Multiplication();
-        dataInstructions.put(multiplication.getKeyword(), multiplication);
```

Comment:
Why those instructions are deleted ?

Reply:
No idea actually, did not delete those on purpose. Probably happened while merging/updating/etc.